containers/ecr-viewer/src/app/view-data/components/PatientBanner.tsx

```diff
@@ -21,9 +21,7 @@ const PatientBanner = ({ bundle, mappings }: PatientBannerProps) => {
   return (
     <div className="patient-banner">
       <span className="patient-banner-name">
-        {bundle && mappings
-          ? evaluatePatientName(bundle, mappings, true)?.[0]
-          : ""}
+        {bundle && mappings ? evaluatePatientName(bundle, mappings, true) : ""}
```

Comment:
I'm pretty sure react will just ignore the undefined and do the empty thing here

```suggestion
        {bundle && mappings && evaluatePatientName(bundle, mappings, true)}
```

Reply:
i dont think we ever want the patient banner to be blank tbh

Reply:
i also think leaving it as it was matches the convention with the date which may have been done on purpose 

`{bundle && mappings
          ? formatDate((evaluate(bundle, mappings.patientDOB) as any[])[0]) ||
            ""
          : ""`

Reply:
agreed 💯 

this logic change doesn't affect that, but maybe we want some other string as a default like `Unknown name` in the off chance there isn't one?

Reply:
seems maybe a bit outside of the scope of this ticket, again all I added was the true for the boolean in this line

Reply:
if I were being very thorough, I'd probably do
```
official > usual > temp > anonymous > nickname > maiden > old
```
but given the rarity of the situation, I'd pragmatically just take the first (and maybe...mayyybeeeee throw a parens with the use after it)